### PR TITLE
MP-6064 Marketplace Product Search Widget feature integration guide

### PR DESCRIPTION
**Pull request by volodymyr-bezuhlyi**
Ticket: https://spryker.atlassian.net/browse/MP-6064

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-product-search-widget-feature-integration.md [hunk 1]
```diff
@@ -0,0 +1,401 @@
+---
+title: Marketplace Product Search Widget feature integration
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-product-search-widget-feature-integration.md:2]: we don't have such a Spryker feature. Please check and update existing IGs according to changes introduced in the epic.

Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-product-search-widget-feature-integration.md:2]: we don't have such a Spryker feature. Please check and update existing IGs according to changes introduced in the epic.

**Review by ilyakubanov — COMMENTED** [review]
Please add `+` features to the corresponding Related features

File: docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-product-quick-add-to-cart-feature-integration.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | INTEGRATION GUIDE        |
+| --------------- | -------- | ------------------ |
+| Spryker Core         | {{page.version}} | [Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/spryker-core-feature-integration.html) |
+| Marketplace Merchant | {{page.version}} | [Marketplace Merchant feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-merchant-feature-integration.html) |
+| Product   | {{page.version}} | [Product feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/product-feature-integration.html) |
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-product-quick-add-to-cart-feature-integration.md]: Not fixed

File: docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-quick-add-to-cart-feature-integration.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+|---------------------------|---------------------------------------------|
+| MerchantProduct           | vendor/spryker/merchant-product             |
+| MerchantProductDataImport | vendor/spryker/merchant-product-data-import |
+| MerchantProductGui        | vendor/spryker/merchant-product-gui         |
+| MerchantProductSearch     | vendor/spryker/merchant-product-search      |
+| MerchantProductStorage    | vendor/spryker/merchant-product-storage     |
+| MerchantProductWidget     | vendor/spryker-shop/merchant-product-widget |
+| QuickOrderPage            | vendor/spryker-shop/quick-order-page        |
+
+{% endinfo_block %}
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-quick-add-to-cart-feature-integration.md:49]: - not needed
- where are plugins?

File: docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-quick-add-to-cart-feature-integration.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+title: Marketplace Product + Quick Add to Cart feature integration
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-quick-add-to-cart-feature-integration.md:2]: must be added to the corresponding Related features sections

**Review by ilyakubanov — COMMENTED** [review]
- new IGs must be in navigation
 
Missing plugins
- MerchantProductOffersSelectWidget
- MerchantProductOfferProductQuickAddFormExpanderPlugin
- MerchantProductOfferProductConcretePageMapExpanderPlugin
- ProductOfferProductConcreteWritePublisherPlugin
- ProductOfferStoreProductConcreteWritePublisherPlugin
- MerchantQuickOrderItemMapperPlugin
- MerchantProductQuickOrderItemExpanderPlugin
- MerchantProductOfferQuickOrderItemExpanderPlugin
- MerchantProductOfferQuickOrderFormColumnPlugin
- MerchantProductOfferQuickOrderFormExpanderPlugin

File: docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-catalog-feature-integration.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+Make sure that the following modules have been installed:
+
+| MODULE          | EXPECTED DIRECTORY              |
+|-----------------|---------------------------------|
+| MerchantProduct | vendor/spryker/merchant-product |
+| Catalog         | vendor/spryker/catalog          |
+
+{% endinfo_block %}
+
+### 4) Set up behavior
```
[the remark this answers is not in the document]
Comment (volodymyr-bezuhlyi) [reply docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-catalog-feature-integration.md:45]: removed row numbers according to IG template `feature-integration-guide-template.md`

File: docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-offer-catalog-feature-integration.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    /**
+     * @return \Spryker\Client\Search\Dependency\Plugin\QueryExpanderPluginInterface[]|\Spryker\Client\SearchExtension\Dependency\Plugin\QueryExpanderPluginInterface[]
+     */
+    protected function createSuggestionQueryExpanderPlugins()
+    {
+        return [
+            new MerchantReferenceQueryExpanderPlugin(),
+        ];
+    }
+}
```
[the remark this answers is not in the document]
Comment (volodymyr-bezuhlyi) [reply docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-offer-catalog-feature-integration.md:86]: Need to discuss this comment

File: docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-offer-feature-integration.md [hunk 7]
```diff
@@ -1444,6 +1453,8 @@ It means the following:
 1. If a merchant gets deactivated, `ProductAbstract`s that were on the catalog search only because they had a product offer from that merchant get removed.
 2. If a product offer gets created, and the `ProductAbstract` related to it was not available on catalog search, it would be available now.
 
+Make sure that merchant product can be found by sku and added to Cart with proper merchant
```
[the remark this answers is not in the document]
Comment (volodymyr-bezuhlyi) [reply docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-product-offer-feature-integration.md:1456]: fixed+moved to Marketplace Product + Marketplace Product Offer feature integration guide

**Review by ilyakubanov — DISMISSED** [review]

**Review by AdvaBD — APPROVED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**